bin/millau/runtime/src/lib.rs

```diff
@@ -582,17 +588,15 @@ generate_bridge_reject_obsolete_headers_and_messages! {
 	BridgeRialtoMessages, BridgeRialtoParachainMessages
 }
 
+bp_runtime::generate_static_str_provider!(BridgeRefundRialtoPara2000Lane0Msgs);
 /// Signed extension that refunds relayers that are delivering messages from the Rialto parachain.
-pub type BridgeRefundRialtoParachainRelayers =
-	bridge_runtime_common::refund_relayer_extension::RefundRelayerForMessagesFromParachain<
-		Runtime,
-		RialtoGrandpaInstance,
-		WithRialtoParachainsInstance,
-		WithRialtoParachainMessagesInstance,
-		RialtoParachainId,
-		RialtoParachainMessagesLane,
-		Runtime,
-	>;
+pub type BridgeRefundRialtoParachainMessages = RefundBridgedParachainMessages<
+	Runtime,
+	RefundableParachain<WithRialtoParachainsInstance, RialtoParachainId>,
+	RefundableMessagesLane<WithRialtoParachainMessagesInstance, RialtoParachainMessagesLane>,
```

Comment:
@serban300 @svyatonik 
I think in the future https://github.com/paritytech/parity-bridges-common/issues/1760,
we will have to read Lanes dynamically (e.g. from onchain storage),
but I guess it is ok for now